docs/intro/tutorial06.txt

```diff
@@ -86,6 +86,10 @@ Start the server (or restart it if it's already running):
 
     $ python manage.py runserver
 
+Note that you must restart the server if it is already running. While the server
+can hot-reload certain changes, like to views or templates, it cannot hot-reload
+changes to static files.
```

Comment:
This is already noted in L83:

_"Start the server (**or restart it if it's already running**):"_

Reply:
It's not noted clearly enough. It's not noted that the server does not pick up changes to static files. There are many other parts of the tutorial that encourage the user to save the file and see the changes without restarting the server. What's the problem with adding some extra text to make the server behavior clearer, especially in a tutorial?

Reply:
Tutorial is not a place to explain everything in detail. Django follows [Diátaxis framework](https://diataxis.fr/tutorials/#ruthlessly-minimise-explanation):

_"A tutorial is not the place for explanation."_